msk-lambda-iam-python-sam/README.md

```diff
@@ -1,99 +1,72 @@
 
-The application creates a Lambda function that listens to Kafka messages on a topic of an MSK Cluster. These resources are defined in the `template.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
+The application creates a Lambda function that listens to Kafka messages on a topic of an MSK Cluster. These resources are defined in the `template_original.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
 
 Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
 
 ## Requirements
 
 * [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
-* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
-* [Git installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
-* Create MSK cluster and topic that will be used for testing. It is important to create the topic before deploying the Lambda function, otherwise the event source mapping will stay disabled. 
 
-## Deploy the sample application
-
-The AWS SAM CLI is a serverless tool for building and testing Lambda applications. It uses Docker to locally test your functions in an Amazon Linux environment that resembles the Lambda execution environment. It can also emulate your application's build environment and API.
-
-To use the AWS SAM CLI, you need the following tools.
-
-* AWS SAM CLI - [Install the AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
-* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
-
-1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
-    ``` 
-    git clone https://github.com/aws-samples/serverless-patterns.git
-    ```
-1. Change directory to the pattern directory:
-    ```
-    cd msk-lambda-iam-python-sam
-    ```
-
-1. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
-    ```
-    sam build
-    sam deploy --guided
-    ```
+## Run the Cloudformation template to create the MSK Cluster and Client EC2 machine
```

Comment:
I'm not going to mark every instance, but please double check the correct spelling:
Cloudformation -> CloudFormation
lambda -> Lambda
EC2 machine -> EC2 instance